### DISCUSSION TITLE
Pilnie poszukuję doświadczonego Azurowca do wynajęcia

### DISCUSSION THREAD

**Discussion by challengerocketcom** [AzureCommunityPL/group#66] [Pomoc]
Poszukuję osoby/firmy, która będzie w stanie przyglądnąć się dokładnie naszej infrastrukturze Azurowej i wyeliminowaniu problemów związanych z wydajnością.
Jest to serwis www oparty o AppSerwisy, ObjectStorage, bazę danych i kilka mniej typowych dodatków.
Zależy mi bardzo na czasie.

Kontakt: mail @ procod . pl

**Comment by wi5nia** [comment]
Ktos sie znalazl bo tak temat wisi i az mi sie nie chce wierzyc, ze nikt sie nie zglosil chetny ;)

**Reply by kamil-mrzyglod** [reply]
Wszyscy zarobieni :P

**Reply by wi5nia** [reply]
Nawet Ty @kamil-mrzyglod ? 😱

**Reply by kamil-mrzyglod** [reply]
Już łatwiej na NFZ do lekarza się umówić 😝

**Reply by ptrstpp950** [reply]
Wiesz jest też 2 problem -> napisz tutaj, że robisz fuchę i już szef do Ciebie pisze: "Zapomniałem ci powiedzieć o taskach 3344, 2343, 5567, 3248, bo czytałem w internecie, że masz trochę wolnego czasu" :)

**Reply by wi5nia** [reply]
Taski? A co jest? Ja myslalem, ze tu same architekty siedzo i nic nie robio... 😲

**Reply by ptrstpp950** [reply]
No niestety ja jestem "ex-architektem" i teraz muszę na życie taskami zarobić. I tak dobrze, że nie taczkami, ale to pewnie już niedługo.

**Reply by wi5nia** [reply]
Zadna praca nie hanbi!

No dobra, jednak tak - bycie coachem hanbi...

**Reply by smereczynski** [reply]
@wi5nia a bycie hackerem hańbi? ;)

**Reply by wi5nia** [reply]
Pytanie czego hakerem... bo "bio" to juz zahacza o to 😄
